src/configs/main.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
file is still in use for two files in production. but is beeing phased out

Reply:
So what's the plan so far?

Reply:
Did a check with VSC. It's gone now